.github/workflows/reference_docs.yml

```diff
@@ -74,8 +75,9 @@ jobs:
           git add .
           git status
           if [[ -z $(git status --porcelain) ]]; then echo "No changes. We're done."; exit 0; fi
-          git commit -m "updated docs for ${branch_specifier}" -a
+          git commit -m "updated docs for ${{ github.event.inputs.branch }}" -a
           git push origin $BRANCH
       - name: Create Pull Request
         run: |
           curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -X POST -d "{\"title\": \"updated docs for ${{ github.event.inputs.branch }}\",\"head\": \"${BRANCH}\",\"base\": \"${{ github.event.inputs.branch }}\"}" https://api.github.com/repos/elastic/logstash-docs/pulls
+      - run: echo "### Updated docs for ${{ github.event.inputs.branch }} :rocket:" >> $GITHUB_STEP_SUMMARY
```

Comment:
cool stuff!

Reply:
Thanks @jsvd for making this workflow happen, it is a tremendous help!